ChangeLog.d/fix-ctr-drbg-may-free-invalid-aes-context.txt

```diff
@@ -0,0 +1,4 @@
+Bugfix
+    * Fix mbedtls_ctr_drbg_init does not call mbedtls_aes_init, which result
+      mbedtls_ctr_drbg_free may call mbedtls_aes_free on an invalid
+      mbedtls_aes_context.
```

Comment:
The changelog entry should mention that this only happens with ALT implementation. Also the fact that this is due to a misplaced call to aes_init is an implementation detail, what matters to users is that some call sequences of ctr_drbg are problematic. Also the sentence is not grammatical. Suggested wording:

> Fix mbedtls_ctr_drbg_free() on an initialized but unseeded context. When MBEDTLS_AES_ALT is enabled, it could call mbedtls_aes_free() on an uninitialized context.